Testscripts/Linux/XDPForwardingTest.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# This script starts pktgen and checks XDP_TX forwarding performance by starting xdpdump application
+# in forwarding configuration on forwarder VM and checks how many packets received at the receiver interface
+# by running xdpdump application in drop configuration (number of packets received == number of packets dropped).
+
+
+packetCount=10000000
+nicName='eth1'
+packetFwdThreshold=90
+
+function convert_MAC_to_HEXArray(){
+        while IFS=':' read -ra ADDR; do
+                size=$((${#ADDR[@]} - 1))
+                MACarr=$(printf '0x%s\n' ${ADDR[$i]})
+                for i in $(seq 1 $size);
+                do
+                        MACarr="$MACarr, $(printf '0x%s\n' ${ADDR[$i]})";
+                done
+        done <<< "$1"
+        echo "$MACarr"
+}
+
+function configure_XDPDUMP_TX(){
+        LogMsg "Configuring TX Setup"
+        # new distros does not have ifconfig present by default
+	LogMsg "Installing net-tools for confirming ifconfig is present in VM."
```

Comment:
indentation? Some lines are 8-char size and others 4-char size.

Reply:
![image](https://user-images.githubusercontent.com/23069399/91615697-fbca4900-e938-11ea-9074-bfbb89c07b9d.png)

Reply:
fixed.